app.py

```diff
@@ -4,12 +4,17 @@
 
 
 if __name__ == '__main__':
-    metrics = metrics.FileMetrics() # this is an observable that streams in all the data alerts->etcd->build
+    metrics_list = ['openshift_build_info', 'etcd_object_counts', 'alerts']
```

Comment:
let's read the list of metrics from an ENV variable. And if that variable exists, then setup `metrics` in the next line

Reply:
Changed it in the new commit. Should I read other variables from env as well (vars like start/end_datetime and chunk_size)?